includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -433,8 +435,46 @@ public function getData() {
 		return $data;
 	}
 
+	private function getSitenameData() {
+		$sitename = $this->wg->Sitename;
+		$wgSitenameForComscoreForWikiId = WikiFactory::getVarValueByName( 'wgSitenameForComscore', $this->wikiId );
+
+		if ( $wgSitenameForComscoreForWikiId ) {
+			$sitename = $wgSitenameForComscoreForWikiId;
+		} else {
+			$wgSitenameForWikiId = WikiFactory::getVarValueByName( 'wgSitename', $this->wikiId );
+
+			if ( $wgSitenameForWikiId ) {
+				$sitename = $wgSitenameForWikiId;
+			}
+		}
```

Comment:
What do you think about using 4th parameter (`$default`) in `getVarValueByName()` to simplify the code?